vitest.config.ts

```diff
@@ -10,7 +10,9 @@ export default defineConfig({
     },
     resolve: {
         alias: {
-            '@': path.resolve(__dirname, '.')
+            '@': path.resolve(__dirname, '.'),
+            '~': path.resolve(__dirname, '.')
```

Comment:
Did it not work if you imported as `@/typedefs/gqlTypes.js`?

Reply:
Yes it did work with the `@`. But since we use the `~` for importing the `/typedefs/gqlTypes.js` in most of the file, I wanted to stay consistent and keep it with the `~`. If that doesn't matter I can change it to `@`